backend/flask/app.py

```diff
@@ -29,7 +29,26 @@ def post_req_handler():
         user = User(email=data['email'], password=encrypted_password)
         db.session.add(user)
         db.session.commit()
-        return jsonify({'message': 'Data saved!'})
+        #return the users id so they can be later identified
```

Comment:
nit space between # and comment

Reply:
thanks for pointing that out, will make the change in my newest pr